DIGNDB.APP.SmitteStop.Jobs/Jobs/EuGateway/UploadTemporaryExposureKeyEuGatewayJob.cs

```diff
@@ -3,15 +3,15 @@
 using Microsoft.Extensions.Logging;
 using System;
 
-namespace DIGNDB.APP.SmitteStop.Jobs.Jobs
+namespace DIGNDB.APP.SmitteStop.Jobs.EuGateway
```

Comment:
This namespace is wrong.
It should correspond to the file location i.e. DIGNDB.APP.SmitteStop.Jobs.Jobs.EuGateway